firetv/__init__.py

```diff
@@ -116,57 +116,215 @@ def connect(self):
                 signer = Signer(self.adbkey)
 
                 # Connect to the device
-                self._adb = adb_commands.AdbCommands().ConnectDevice(serial=self.host, rsa_keys=[signer])
+                self.adb = adb_commands.AdbCommands().ConnectDevice(serial=self.host, rsa_keys=[signer])
```

Comment:
While we're at it, we should refactor this part along these lines:

```
kwargs = {
    'serial': self.host
}
if self.adbkey:
    kwargs['rsa_keys'] = [Signer[self.key]]
self.adb = adb_commands.AdbCommands().ConnectDevice(**kwargs)
```

Reply:
Fixed in https://github.com/JeffLIrion/python-firetv/commit/1a484423d522fe0780e4d485fa968c9eff1bfc94